.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - 5.4
```

Comment:
Better to change this to 5.5.9, so we'r testing on the lowest supported version too.